grunt/configs/shell.js

```diff
@@ -1,19 +1,30 @@
 /* jshint node:true */
-module.exports = function( grunt ) {
+module.exports = function () {
 	'use strict';
 
-	var config = {};
+	let headers = {
+		"Report-Msgid-Bugs-To": "https://wordpress.org/support/plugin/wc-plugin-framework",
```

Comment:
Since this isn't a repo plugin, should this be something else? Maybe:

```suggestion
		"Report-Msgid-Bugs-To": "https://github.com/godaddy-wordpress/wc-plugin-framework/issues",
```

Reply:
Oh yeah that makes perfect sense